modules/oadp-configuring-aws-md5sum.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-aws-md5sum_{context}"]
+= Configuring the backup storage location with MD5 checksum algorithm
+
+You can configure the Backup Storage Location (BSL) in the Data Protection Application (DPA) to use a MD5 checksum algorithm for {aws-short} S3 and S3 compatible storage providers. The checksum algorithm calculates the checksum for uploading and downloading objects to S3. You can use one of the following options to set the `checksumAlgorithm` field in the `spec.backupLocations.velero.config.checksumAlgorithm` section of the DPA.
```

Comment:
```suggestion
You can configure the Backup Storage Location (BSL) in the Data Protection Application (DPA) to use a MD5 checksum algorithm for both Amazon Simple Storage Service (Amazon S3) and S3-compatible storage providers. The checksum algorithm calculates the checksum for uploading and downloading objects to Amazon S3. You can use one of the following options to set the `checksumAlgorithm` field in the `spec.backupLocations.velero.config.checksumAlgorithm` section of the DPA.
```
Hm, it appears that Amazon calls this "Amazon Simple Storage Service (Amazon S3)", see https://aws.amazon.com/s3/